local_pathfinding/coord_systems.py

```diff
@@ -45,6 +45,22 @@ def cartesian_to_true_bearing(cartesian: float) -> float:
     return (90 - cartesian + 360) % 360
 
 
+def knots_to_kilometers_per_hour(knots: float) -> float:
```

Comment:
Although trivial, we should add tests for each function

Reply:
Also is this function even used anywhere?

Reply:
No, it used to be. But if speeds can be published in kmph its not needed